pinot-common/src/test/java/com/linkedin/pinot/common/segment/fetcher/SegmentFetcherFactoryTest.java

```diff
@@ -57,26 +57,35 @@ public void testInitSegmentFetcherFactory() throws Exception {
     Assert.assertEquals(((TestSegmentFetcher)SegmentFetcherFactory.getPreloadSegmentFetchers().get("test")).init_called, 1);
     ReplacedFetcher fetcher = (ReplacedFetcher)SegmentFetcherFactory.getPreloadSegmentFetchers().get(replacableProto);
     Assert.assertEquals(fetcher.getInitCalled(), 1);
+    SegmentFetcherFactory.getPreloadSegmentFetchers().clear();
```

Comment:
Should the clear be at the beginning of test? That would avoid any new tests that forgets to clear it at the end?

Reply:
OK